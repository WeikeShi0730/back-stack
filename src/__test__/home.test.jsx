import { render, screen } from "@testing-library/react";
import Home from "../pages/home/home.component";

describe("<Home />", () => {
  test("<Home />", () => {
    const { baseElement } = render(<Home />);
    expect(screen.getByText("Back Stack")).toBeInTheDocument();
    expect(baseElement).toMatchSnapshot();
  });
});
