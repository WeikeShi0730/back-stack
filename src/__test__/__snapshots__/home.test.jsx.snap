// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> <Home /> 1`] = `
<body>
  <div>
    <div
      class="h-full flex m-auto"
    >
      <div
        class="m-auto p-5 flex flex-col flex-grow font-light text-sm md:text-base lg:text-lg"
      >
        <div>
          <p
            class="text-7xl"
          >
            Back Stack
          </p>
          <br />
          Tracker of
           
          <span
            class="underline decoration-green-700 underline-offset-1 text-lg md:text-xl lg:text-2xl"
          >
            spine alignment
          </span>
          , focusing on rehabilitating stroke vicims.
        </div>
        <br />
        <div>
          <p>
            Losing track of your position? There is something you can do about it.
          </p>
        </div>
        <br />
        <br />
        <br />
        <br />
        <div>
          <p>
            Back Stack works in
             
            <span
              class="underline decoration-green-700 underline-offset-1 text-lg md:text-xl lg:text-2xl"
            >
              3
            </span>
             
            easy steps:
          </p>
          <ul>
            <br />
            <li>
              1. Place Stack on your back.
            </li>
            <br />
            <li>
              2. Get real-time feedback for instant correction and track your progress via the mobile app.
            </li>
            <br />
            <li>
              3. Get your personal analysis and progress report.
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>
</body>
`;
